jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -654,7 +677,158 @@ def _set_value_in_subset_definition(self, index, name, desired_key, new_value):
                 self.subset_definitions[index][i]['value'] = new_value
                 break
 
-    def _import_spectral_regions(self, spec_region, mode=NewMode, viewer=None):
+    def import_region(self, region, **kwargs):
+        return_bad_regions = kwargs.pop('return_bad_regions', None)
```

Comment:
Why all the popping? Why not define these keywords explicitly instead of a blind `**kwargs`? Given that this is a public API, it is important for users to know all their options in the docstring as well (when you write the docstring that Kyle asked for).